### PR TITLE
Feature/issue 1 basic structure

### PR DESCRIPTION
**Pull request by andreit**

### CODE REVIEW

**Comment by andreit** [comment]
Fixes #1

**Review by Peter-TeamGeek — APPROVED** [review]

File: src/flask_odoo/__init__.py [hunk 1]
```diff
@@ -1,5 +1,159 @@
...
+    def __getattr__(self, name):
+        return Method(name, self)
+
+    def __repr__(self):
+        return (
+            f"<Model(name='{self.name}', proxy={self.proxy}, db='{self.db}')>"
+        )
+
+
+class Method:
```
Comment (EduanMullerTG) [inline src/flask_odoo/__init__.py:137]: I love the way these work, this is some python magic right here.

**Review by EduanMullerTG — APPROVED** [review]

File: src/flask_odoo/__init__.py (hunk 1 above)
Comment (andreit) [reply src/flask_odoo/__init__.py:137]: The `__xxx__` methods are actually referred to as "magic" methods 😉

File: src/flask_odoo/__init__.py [hunk 2]
```diff
@@ -1,5 +1,159 @@
...
+        ...                                         raise_exception=False)
+        true
+
+    """
+
+    def __init__(self, name, model):
+        self.name = name
+        self.model = model
+
+    def __call__(self, *args, **kwargs):
```
Comment (sluipmoord) [inline src/flask_odoo/__init__.py:155]: 🧙

**Review by sluipmoord — APPROVED** [review]

**Pull request merged**